docs/hopsml/experiment.rst

```diff
@@ -285,7 +301,8 @@ After launching your job using experiment, you can navigate to Hopsworks to view
 Experiments service
 -------------------
 
-Experiments service provides a unified view of all the experiments run using the experiment API in the hops python library. As demonstrated in the picture it provides general information about the experiment and the resulting metric.
+Experiments service provides a unified view of all the experiments run using the experiment API in the hops python library. As demonstrated in the picture it provides general information about the experiments and provides the possibility to compare hyperparameter and metrics to find the best model.
+Users do not need to register experiments themselves in any explicit way, it is enough to use any of the `experiment` module's functions for the experiment to appear.
```

Comment:
You could be a bit more explicit here.
it is enough to just called `experiment.launch(..)` or any of the other `experiment` module's functions for the experiment results to logged.